### ISSUE TITLE
Bug report: Parentheses after ellipsis?

### ISSUE THREAD

**Issue by rob-legrand** [douglascrockford/JSLint#481]
**Describe the bug**
1. JSLint sometimes doesn't like parentheses after ellipsis, as illustrated below.  My humble guess is that this was unintentional.

2. JSLint edition v2025.10.31

3. Code snippet:

```javascript
let f;
f(f);
f(f, (f));
f(f, (f), f);
f((f), f);
f((f));      // line A (should warn)
f(...f);
f(f, ...(f));
f(f, ...(f), f);
f(...(f), f);
f(...(f));   // line B (should pass?)
```

**Expected behavior**
JSLint warns on both lines A and B.  I agree for line A, but I think line B should pass.  It is sometimes useful to be able to use parentheses after ellipsis, such as when a function is called with a number of parameters that varies with the situation.  Example:
```javascript
newElement.replaceChildren(...(
    Array.isArray(newChildren)
    ? newChildren
    : [document.createTextNode("Howdy!")]
));
```
Such usage neatly avoids an if statement.
A suggested change that may fix it to satisfaction:
from:
```javascript
if (the_argument.wrapped === true) { // line 4677 in jslint.mjs
```
to:
```javascript
if (the_argument.wrapped === true && the_argument.ellipsis !== true) {
```

**Comment by kaizhu256** [comment]
- @rob-legrand, your fix looks ok.
- can you create a pull-request to jslint's `beta` branch?
  - happy to attribute it to you, and guide in writing test-cases and code-coverage for the pull too!
